lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
Yeh, I was just thinking that some people might get confused and think it is the actual dropdown toggle button though.